vod/add-captions.md

```diff
@@ -15,13 +15,696 @@ The captions are uploaded per language. You can upload different captions for ea
+| English (Vanuatu) | en-VU | English (Vanuatu) |
+| English (Samoa) | en-WS | English (Samoa) |
+| English (South Africa) | en-ZA | English (South Africa) |
+| English (Zambia) | en-ZM | English (Zambia) |
+| English (Zimbabwe) | en-ZW | English (Zimbabwe) |
+| Esperanto | eo | Esperanto |
+| Esperanto (world) | eo-001 | Esperanto (mondo) |
+| Spanish | es | Español |
+| Spanish (Latin America) | es-419 | Español (Latinoamérica) |
+| Spanish (Argentina) | es-AR | Español (Argentina) |
+| Spanish (Bolivia) | es-BO | Español (Bolivia) |
+| Spanish (Brazil) | es-BR | Español (Brasil) |
+| Spanish (Belize) | es-BZ | Español (Belice) |
+| Spanish (Chile) | es-CL | Español (Chile) |
+| Spanish (Colombia) | es-CO | Español (Colombia) |
+| Spanish (Costa Rica) | es-CR | Español (Costa Rica) |
+| Spanish (Cuba) | es-CU | Español (Cuba) |
+| Spanish (Dominican Republic) | es-DO | Español (República Dominicana) |
+| es-EA | es-EA | es-EA |
+| Spanish (Ecuador) | es-EC | Español (Ecuador) |
+| Spanish (Spain) | es-ES | Español (España) |
+| Spanish (Equatorial Guinea) | es-GQ | Español (Guinea Ecuatorial) |
+| Spanish (Guatemala) | es-GT | Español (Guatemala) |
+| Spanish (Honduras) | es-HN | Español (Honduras) |
+| es-IC | es-IC | es-IC |
+| Spanish (Mexico) | es-MX | Español (México) |
+| Spanish (Nicaragua) | es-NI | Español (Nicaragua) |
+| Spanish (Panama) | es-PA | Español (Panamá) |
+| Spanish (Peru) | es-PE | Español (Perú) |
+| Spanish (Philippines) | es-PH | Español (Filipinas) |
+| Spanish (Puerto Rico) | es-PR | Español (Puerto Rico) |
+| Spanish (Paraguay) | es-PY | Español (Paraguay) |
+| Spanish (El Salvador) | es-SV | Español (El Salvador) |
+| Spanish (United States) | es-US | Español (Estados Unidos) |
+| Spanish (Uruguay) | es-UY | Español (Uruguay) |
+| Spanish (Venezuela) | es-VE | Español (Venezuela) |
+| Estonian | et | Eesti |
+| Estonian (Estonia) | et-EE | Eesti (Eesti) |
+| Basque | eu | Euskara |
+| Basque (Spain) | eu-ES | Euskara (Espainia) |
+| Persian | fa | فارسی |
+| Persian (Afghanistan) | fa-AF | فارسی (افغانستان) |
+| Persian (Iran) | fa-IR | فارسی (ایران) |
+| Fula | ff | Pulaar |
+| Fula (Adlam) | ff-Adlm | 𞤆𞤵𞤤𞤢𞤪 (𞤀𞤁𞤂𞤢𞤃) |
+| Fula (Adlam, Burkina Faso) | ff-Adlm-BF | 𞤆𞤵𞤤𞤢𞤪 (𞤀𞤁𞤂𞤢𞤃⹁ 𞤄𞤵𞤪𞤳𞤭𞤲𞤢 𞤊𞤢𞤧𞤮𞥅) |
+| Fula (Adlam, Cameroon) | ff-Adlm-CM | 𞤆𞤵𞤤𞤢𞤪 (𞤀𞤁𞤂𞤢𞤃⹁ 𞤑𞤢𞤥𞤢𞤪𞤵𞥅𞤲) |
+| Fula (Adlam, Ghana) | ff-Adlm-GH | 𞤆𞤵𞤤𞤢𞤪 (𞤀𞤁𞤂𞤢𞤃⹁ 𞤘𞤢𞤲𞤢) |
+| Fula (Adlam, Gambia) | ff-Adlm-GM | 𞤆𞤵𞤤𞤢𞤪 (𞤀𞤁𞤂𞤢𞤃⹁ 𞤘𞤢𞤥𞤦𞤭𞤴𞤢) |
+| Fula (Adlam, Guinea) | ff-Adlm-GN | 𞤆𞤵𞤤𞤢𞤪 (𞤀𞤁𞤂𞤢𞤃⹁ 𞤘𞤭𞤲𞤫) |
+| Fula (Adlam, Guinea-Bissau) | ff-Adlm-GW | 𞤆𞤵𞤤𞤢𞤪 (𞤀𞤁𞤂𞤢𞤃⹁ 𞤘𞤭𞤲𞤫-𞤄𞤭𞤧𞤢𞤱𞤮𞥅) |
+| Fula (Adlam, Liberia) | ff-Adlm-LR | 𞤆𞤵𞤤𞤢𞤪 (𞤀𞤁𞤂𞤢𞤃⹁ 𞤂𞤢𞤦𞤭𞤪𞤭𞤴𞤢𞥄) |
+| Fula (Adlam, Mauritania) | ff-Adlm-MR | 𞤆𞤵𞤤𞤢𞤪 (𞤀𞤁𞤂𞤢𞤃⹁ 𞤃𞤮𞤪𞤼𞤢𞤲𞤭𞥅) |
+| Fula (Adlam, Niger) | ff-Adlm-NE | 𞤆𞤵𞤤𞤢𞤪 (𞤀𞤁𞤂𞤢𞤃⹁ 𞤐𞤭𞥅𞤶𞤫𞤪) |
+| Fula (Adlam, Nigeria) | ff-Adlm-NG | 𞤆𞤵𞤤𞤢𞤪 (𞤀𞤁𞤂𞤢𞤃⹁ 𞤐𞤢𞤶𞤫𞤪𞤭𞤴𞤢𞥄) |
+| Fula (Adlam, Sierra Leone) | ff-Adlm-SL | 𞤆𞤵𞤤𞤢𞤪 (𞤀𞤁𞤂𞤢𞤃⹁ 𞤅𞤢𞤪𞤢𞤤𞤮𞤲) |
+| Fula (Adlam, Senegal) | ff-Adlm-SN | 𞤆𞤵𞤤𞤢𞤪 (𞤀𞤁𞤂𞤢𞤃⹁ 𞤅𞤫𞤲𞤫𞤺𞤢𞥄𞤤) |
```

Comment:
The encoding looks messed up for some languages, but it's coming from the list I provided, I need to look into that!

Reply:
Looks like a problem on my computer as it is fine for other people, I think we're good!